### PR TITLE
goenv 20161028 (new formula)

### PR DESCRIPTION
**Pull request by josegonzalez**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: goenv.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  desc "Go version management"
+  homepage "https://github.com/syndbg/goenv"
+  url "https://github.com/syndbg/goenv/archive/20161028.tar.gz"
+  sha256 "3f283537c2b4f64a7549a009361ac4da6115be6b2c45462b008a58f8366a8804"
+  head "https://github.com/syndbg/goenv.git"
+
+  bottle :unneeded
+
+  def install
+    inreplace "libexec/goenv", "/usr/local", HOMEBREW_PREFIX
```
Comment (MikeMcQuaid) [inline goenv.rb]: Can you please submit this patch to the upstream developers of this project asking for a better way to configure this and add a link to the upstream patch submission and as a comment in the formula file. Thanks!

Comment (josegonzalez) [inline goenv.rb]: I can try that, but this formula is simply following the pattern set by the `pyenv` and `rbenv` formula.

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

File: goenv.rb (hunk 1 above)
Comment (MikeMcQuaid) [inline goenv.rb]: Probably have a `configure` script or similar.

**Pull request merged**